fix error in test run section (#4175)
circleci/circleci-docs: jekyll/_cci2/parallelism-faster-jobs.md

bundle exec rspec $(cat /tmp/tests-to-run)
```

> The file `/tmp/tests-to-run` will have different contents in each container, based on `$CIRCLE_NODE_INDEX` and `$CIRCLE_NODE_TOTAL`.

Author: ```suggestion
The contents of the file `/tmp/tests-to-run` will be different for each container, based on `$CIRCLE_NODE_INDEX` and `$CIRCLE_NODE_TOTAL`.
```

> The contents of the file `/tmp/tests-to-run` will be different for each container, based on `$CIRCLE_NODE_INDEX` and `$CIRCLE_NODE_TOTAL`.

Reviewer: ~ Maybe say "different in each container"?

> The contents of the file `/tmp/tests-to-run` will be different for each container, based on `$CIRCLE_NODE_INDEX` and `$CIRCLE_NODE_TOTAL`.

Author: ```suggestion
The contents of the file `/tmp/tests-to-run` will be different in each container, based on `$CIRCLE_NODE_INDEX` and `$CIRCLE_NODE_TOTAL`.
```
